src/pages/App.js

```diff
@@ -38,7 +40,13 @@ export default function App() {
 
   return (
     <div>
-      <h1> Find Out Best Overall Diets</h1>
+      <h1> BotTrition</h1>
+      <Row>
+        <AnchorButton href="/profile" text="View Profile" />
```

Comment:
The React Router addition changed the `href` prop in `AnchorButton` to `to`. Changing `href` to `to` should be the last change necessary to ensure this works with the newest changes. It may be a good idea to pull from main as well, to ensure the test cases pass.

Reply:
I am going to update all AnchorButton hrefs -> to and then commit again to verify that this was done properly.